src/util/CardsData.tsx

```diff
@@ -0,0 +1,38 @@
+export let CardData = [
```

Comment:
`const CardData`

Reply:
did not take into account in this correction, but to will be in next PR